repositories/src/main/scala/cromwell/pipeline/datastorage/dto/TypedValue.scala

```diff
@@ -23,7 +23,7 @@ object TypedValue {
     implicit val floatTyped: OFormat[FloatTyped] = Json.format
     implicit val booleanTyped: OFormat[BooleanTyped] = Json.format
 
-    adtFormat("$type")(
+    adtFormat("_type")(
```

Comment:
what do this mean? could you explain? any regexp?

Reply:
Of course, you can show conversations on https://github.com/EpamLifeSciencesTeam/cmwl_pipeline/pull/84#discussion_r469159503